src/source/Signaling/LwsApiCalls.c

```diff
@@ -753,6 +768,10 @@ STATUS describeChannelLws(PSignalingClient pSignalingClient, UINT64 time)
 
 CleanUp:
 
+    if (STATUS_FAILED(retStatus)) {
```

Comment:
There was a macro I believe called CHK_LOG_ERR or something similar.

Reply:
I will make this change as part of an upcoming PR.